templates/zerver/help/configure-mobile-notifications.md

```diff
@@ -0,0 +1,22 @@
+# Configure Mobile Push Notifications
+
+Follow these steps to change your mobile notification settings.
+
+{!go-to-the.md!} [Notifications](/#settings/notifications)
+{!settings.md!}
+
+    Under the **Private messages and @-mentions** section check the
+    box by **Mobile push notifications when offline**  if you want
+    mobile notifications only when you're offline.
+
+    ![Mobile Notifications
+    Offline](/static/images/help/mobile-notifications-offline.png)
```

Comment:
We're trying to keep it similar to http://localhost:9991/help/configure-audible-notifications